e2e/app.e2e.js

```diff
@@ -1,136 +1,134 @@
-  beforeEach(function() {
-    return this.app.client.windowByIndex(0);
-  });
-  afterEach(testUtils.afterEach);
-
-  it('shows one window', function() {
-    return this.app.browserWindow.isVisible().then(function(visible) {
-      assert.equal(visible, true);
-    }).getWindowCount().then(function(count) {
-      assert.equal(count, 1);
-    });
-  });
-
-  it('shows welcome screen', function() {
-    return this.app.client.element('.welcome').isVisible().then(function(visible) {
-      assert.equal(visible, true);
-    });
-  });
-
-  describe('when clicking to add file', function() {
-    beforeEach(function() {
-      return this.app.client.click('#plus-button-open').waitForVisible('.table .body .row', 1000);
-    });
-
-    it('should show file', function() {
-      var app = this.app;
-
+/* eslint-env mocha */
+import { assert } from 'chai'
+import testUtils from './utils'
+
+describe('application launch', function () {
+  beforeEach(testUtils.beforeEach)
+  beforeEach(function () {
+    return this.app.client.windowByIndex(0)
+  })
+  afterEach(testUtils.afterEach)
+
+  it('shows one window', function () {
+    return this.app.browserWindow.isVisible().then(function (visible) {
+      assert.equal(visible, true)
+    }).getWindowCount().then(function (count) {
+      assert.equal(count, 1)
+    })
+  })
+
+  it('shows welcome screen', function () {
+    return this.app.client.element('.welcome').isVisible().then(function (visible) {
+      assert.equal(visible, true)
+    })
+  })
+
+  describe('when clicking to add file', function () {
+    beforeEach(function () {
+      return this.app.client.click('#plus-button-open').waitForVisible('.table .body .row', 1000)
+    })
+
+    it('should show file', function () {
       return this.app.client.element('.table .body .row:nth-child(1)')
-        .isVisible().then(function(visible) {
-          assert.equal(visible, true);
+        .isVisible().then(function (visible) {
+          assert.equal(visible, true)
         })
         .waitUntilTextExists('.table .body .row .cell:nth-child(2)', '2016-06-13')
-        .getText('.table .body .row').then(function(text) {
-          assert.include(text, 'readable.pdf');
-          assert.include(text, '2016-06-13');
-          assert.include(text, '6000.00');
-        });
-    });
-
-    describe('when copying', function() {
-      it('should copy CSV to clipboard', function() {
+        .getText('.table .body .row').then(function (text) {
+          assert.include(text, 'readable.pdf')
+          assert.include(text, '2016-06-13')
+          assert.include(text, '6000.00')
+        })
+    })
+
+    describe('when copying', function () {
+      it('should copy CSV to clipboard', function () {
         return this.app.client.click('.table .body .row')
-          .execute(function() {
-            document.dispatchEvent(new Event('copy'));
+          .execute(function () {
+            document.dispatchEvent(new window.Event('copy'))
```

Comment:
Issue [562](https://github.com/feross/standard/issues/562) and [619](https://github.com/feross/standard/pull/619)
we can go either way `window.` or a comment `/* global */`